dev/source/docs/ros2-gazebo.rst

```diff
@@ -18,11 +18,11 @@ We will clone the required repositories using `vcstool <https://github.com/dirk-
     wget https://raw.githubusercontent.com/ArduPilot/ardupilot_gz/main/ros2_gz.repos
     vcs import --recursive src < ros2_gz.repos
 
-Set the gazebo version
+Set the gazebo version to either ``garden`` or ``harmonic``
 
 .. code-block:: bash
 
-    export GZ_VERSION=garden
+    export GZ_VERSION=harmonic
```

Comment:
I'm confident to recommend using `harmonic` as default in macOS, may need additional testing on Ubuntu before switching?

Reply:
Yea, I'll leave at garden default for now.